glommio/src/io/read_result.rs

```diff
@@ -95,9 +95,6 @@ impl ReadResultInner {
         }
     }
 
-    /// Creates a slice of this ReadResult with the given offset and length.
```

Comment:
they are unnecessary, but don't hurt either. Maybe we can leave them in place with `//`

Reply:
which? The `slice*` methods?

Reply:
the comments, I mean.
Doc comments may not be necessary, but comments are helpful and there is no harm in leaving them be.

Overall the series looks very good.

Reply:
I didn't want to remove the doc comments as then `clippy` would complain about missing doc comments...